lib/Mongo/MongoGridFS.php

```diff
@@ -48,14 +56,34 @@ class MongoGridFS extends MongoCollection {
      * @param mixed $chunks  [optional]
      * @return MongoGridFS
      */
-    public function __construct($db, $prefix = "fs", $chunks = "fs") {}
+    public function __construct(MongoDB $db, $prefix = "fs", $chunks = null)
+    {
+        if ($chunks) {
+            trigger_error(E_DEPRECATED, "The 'chunks' argument is deprecated and ignored");
+        }
+        if (empty($prefix)) {
+            throw new \InvalidArgumentException('prefix can not be empty');
+        }
+
+        $this->database = $db;
+        $this->filesName = $prefix . '.files';
```

Comment:
You should declare these properties instead of defining them implicitely (and thus public)